ConcernsCaseWork/ConcernsCaseWork/Models/CaseActions/SRMA.cs

```diff
@@ -5,8 +5,15 @@ namespace ConcernsCaseWork.Models.CaseActions
 {
 	public class SRMA : CaseAction
```

Comment:
Could you change the name of this class to SRMAModel

Reply:
pushed the suggested change